x-pack/plugins/lens/public/editor_frame_service/editor_frame/config_panel/chart_switch/chart_switch_selectable.tsx

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React from 'react';
+import { EuiSelectable, EuiSelectableOption, EuiSelectableProps, IconType } from '@elastic/eui';
```

Comment:
Add `EuiPopoverTitle` import.

```suggestion
import { EuiPopoverTitle, EuiSelectable, EuiSelectableOption, EuiSelectableProps, IconType } from '@elastic/eui';
```